package.json

```
@@ -40,6 +40,7 @@
   "dependencies": {
     "@aws-cdk/assert": "~1.74.0",
     "@aws-cdk/aws-autoscaling": "~1.74.0",
+    "@aws-cdk/aws-cloudwatch-actions": "~1.74.0",
```

This'll need rebasing from main since https://github.com/guardian/cdk/pull/191.

```suggestion
    "@aws-cdk/aws-cloudwatch-actions": "1.86.0",
```